src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -2658,3 +2659,25 @@ void java_bytecode_convert_method(
 
   java_bytecode_convert_method(class_symbol, method);
 }
+
+const bool java_bytecode_convert_methodt::is_method_inherited(
+    const irep_idt &classname, const irep_idt &methodid) const
+{
+  class_hierarchyt ch;
+  ch(symbol_table);
+  const auto &parents=ch.get_parents_trans(classname);
+  std::string stripped_methodid=id2string(methodid);
+  stripped_methodid.erase(0, classname.size());
+  for(const auto &parent : parents)
+  {
+    const irep_idt id=id2string(parent)+stripped_methodid;
+    const auto it=symbol_table.symbols.find(id);
+    if(it!=symbol_table.symbols.end())
+    {
+      if(it->second.type.id()==ID_code &&
+         it->second.type.get(ID_access)!=ID_private)
```

Comment:
if the classes are not in the same package, then `ID_default` is also not visible